CHANGELOG.rst

```diff
@@ -9,6 +9,10 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 **Added**
 
+* Increase Space allocated for product description in Offer PDF (`#675 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/675>`_)
+
+* Allocate space for each productItem dependent on number of characters in each property (`#675 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/675>`_)
+
```

Comment:
Since this is one issue and one PR that solves it it should be one point in the CL (NIT)

```suggestion
* Increase space allocated for product description in Offer PDF based on the number of characters (`#675 <https://github.com/qbicsoftware/offer-manager-2-portlet/issues/675>`_)

```